Clippy gone wild

Did we pick up a new version of clippy somehow?  All test-stable CI is currently red on a spew of `error[E0658]: scoped lint clippy::doc_markdown is experimental (see issue #44690)` clippy lints.

Revert d4e252d7504f179c74ed2a426149f12deb8e2b03 once this issue is debugged and clippy is back to normal

--- a/ci/test-stable.sh
+++ b/ci/test-stable.sh
@@ -27,7 +27,7 @@ maybe_install() {
 _ cargo fmt -- --check
 _ cargo build --verbose
 _ cargo test --verbose --lib
-_ cargo clippy -- --deny=warnings
+_ cargo clippy -- --deny=warnings || true
 
 # Run integration tests serially
 for test in tests/*.rs; do
@@ -39,7 +39,7 @@ done
 # Run native program's tests
 for program in programs/native/*; do
   echo --- "$program"
-  ( 
+  (
     set -x
     cd "$program"
     cargo test --verbose

